specification/healthcareapis/resource-manager/Microsoft.HealthcareApis/preview/2022-07-01-preview/healthcare-apis.json

```diff
@@ -4093,6 +4138,224 @@
         "modelAsString": true
       },
       "readOnly": true
+    },
+    "MedtechMeasurementProperty": {
+      "type": "object",
+      "description": "The values for an extracted measurement",
+      "properties": {
+        "name": {
+          "description": "The name of the measurement",
+          "type": "string"
+        },
+        "value": {
+          "description": "The value of the measurement",
+          "type": "string"
+        }
+      },
+      "readOnly": true
+    },
+    "MedtechMeasurement": {
+      "type": "object",
+      "description": "A measurement extracted during the Medtech device event normalization process.",
+      "properties": {
+        "type": {
+          "description": "The type of measurement being collected.",
+          "type": "string"
```

Comment:
can this be an enum ?

Reply:
Unfortunately not. This value is supplied by our end users and can be any string.